src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ExecutionUtils.java

```diff
@@ -443,28 +442,26 @@ public void writeCsvResponse(Result[] resultSet, HttpServletResponse servletResp
   public static <K extends Comparable<K> & Serializable, V extends Number>
       SortedMap<OSHDBCombinedIndex<OSHDBTimestamp, K>, V> computeResult(
       RequestResource requestResource,
-      MapAggregator<OSHDBCombinedIndex<OSHDBTimestamp, K>, OSMEntitySnapshot> preResult)
+      MapAggregator<OSHDBCombinedIndex<OSHDBTimestamp, K>, OSMEntitySnapshot> mapAgg)
```

Comment:
I'm not 100% sure, but shouldn't these two lines be indented one level more, as they are parameters to an additional function?

Reply:
At first I'd thought your are right. However, checkstyle doesn't complain about it, so maybe both options are valid? Also, since some other long functions are formatted this way in the ohsome-api I'd rather not touch them in this PR. :shrug: